server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -1058,6 +1064,38 @@ func (s *BuildBuddyServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// TODO(siggisim): Figure out why this JWT is overriding authority auth and remove.
 	ctx := context.WithValue(r.Context(), "x-buildbuddy-jwt", nil)
 
+	if lookup.Filename[0:2] == "//" {
```

Comment:
this is slightly panic inducing

![image](https://user-images.githubusercontent.com/141737/214359521-5b1e3ba6-8f88-45cf-8af8-983bb4f50357.png)

Could we use strings.HasPrefix / strings.TrimPrefix (below) instead?

Reply:
haha, absolutely. Great idea.